esmvalcore/_recipe.py

```diff
@@ -584,9 +583,9 @@ def _get_ancestors(variable, config_user):
     (input_files, dirnames,
      filenames) = _get_input_files(variable, config_user)
 
-    logger.info("Using input files for variable %s of dataset %s:\n%s",
-                variable['short_name'], variable['dataset'],
-                '\n'.join(input_files))
+    logger.debug("Using input files for variable '%s' of dataset %s:\n%s",
+                 variable['short_name'], variable['alias'].replace('_', ' '),
+                 '\n'.join(input_files))
```

Comment:
How long does it take for you? For me, there is never more than a few seconds without output while still parsing the recipe. Once computations have started there is less output, maybe that is something to look into.

Reply:
with a large list of datasets in a recipe and a saturated and slow JASMIN node it can take order 10min or more - that is, while it's still looking and finding data, so if we don't have a progress bar or some advancing dots or a rotating bar or some elf for Xmas, it'll be a while w/o screen output. Don't think that'll kill the CI job (that was what, one hour w/o screen output right?), but it's quite disconcerting for the user, it feels like the thing went to software heavens

Reply:
It also depends on how the data is stored. Jasmin ERA5, for example, is awfully slow here because of the humongous amount of folders and files that it uses

Reply:
I added an info message that says the that the files were found, but without listing all of them on the screen. The complete list of files is still available in the debug log.

Reply:
OK, but the cursor pointer will still be blinking and blinking while the files are being found with no output to screen? That's OK, let's see if that causes any issues (complaints from users) and we can change it to a moving progress bar, or a walking turtle or something, in the future